stream_alert/apps/_apps/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Import some package level items to make implementing subclasses a bit nicer"""
```

Comment:
The path `stream_alerts/apps/_apps` seems potentially confusing. The `_apps` folder is for actual app instantiations, right? Maybe `stream_alert/apps/_integrations`?

It does distinguish itself with the underscore, so I'm happy with whatever you think is best